src/main/java/no/nav/tag/kontakt/oss/KontaktskjemaController.java

```diff
@@ -24,7 +28,43 @@ public ResponseEntity meldInteresse(
         if (kontaktskjemaService.harMottattForMangeInnsendinger()) {
             return ResponseEntity.status(HttpStatus.TOO_MANY_REQUESTS).build();
         }
+
+        if (!erGyldigKontaktskjema(kontaktskjema)) {
```

Comment:
En klassiker ifm. validering er om valideringsmetoden bør returnere true/false (som man så må sjekke på) eller om den skal være void og kaste en exception ved valideringsfeil. Det siste er ofte litt mer elegant, da man kan forutsette at validering er ok og slipper masse if/else. Hvis du kaster exception slipper du også å returnere det boolske uttrykket som and-er sammen alle valideringene av enkeltfeltene lenger nede.